StoreApp/Controllers/AccountController.cs

```diff
@@ -156,7 +157,7 @@ public ActionResult EditUser(UserEditViewModel userEdit)
             if (userValidateService.CheckForEditUser(userDTO))
             {
                 userService.Edit(userDTO);
-
+                TempData["StatusMessage"] = "success";
                 TempData["Message"] = "Your edit account successful!";
```

Comment:
It's better to write
"The user was updated successfully"

Reply:
fixed